facility-manager.html

```diff
@@ -0,0 +1,806 @@
+</div> <!-- end #line-block -->
+
+<img alt="../images/facilityManagerScreenInsight.jpg" class="align-center" src="images/facilityManagerScreenInsight.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+</ol>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3>OMERO.web</h3>
+
+<ol>
+
+<li>
+
+<p class="first">Click on <span class="bold">New Project</span>, <span class="bold">New Dataset</span> or <span class="bold">New Container</span> icon or <span class="bold">Right-click</span> in the data tree to create a new Project, Dataset or Container.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/facilityManagerNewWeb.jpg" class="align-center" src="images/facilityManagerNewWeb.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+</ol>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="measure"></a>Measurement and analysis</h2>
+
+<p>OMERO.insight has a Measurement Tool that enables Regions of Interest (ROIs) to be drawn on images, and then basic intensity analysis to be performed. Details on how to use it are in the <a href="http://help.openmicroscopy.org/measurement-tool.html" target="_blank" class="external-url">ROIs and Measuring Tool</a> section of the Help.</p>
+
+<p>There are a number of standard scripts shipped with OMERO, and the Administrator can add additional scripts to the repository so they run on the OMERO server and are available in the scripts menu of both clients.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/facilityManagerScripts.jpg" class="align-center" src="images/facilityManagerScripts.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+
+<p>More complex analysis can be done using the OMERO API that allows scripts written in Python, Matlab and other languages to use data from the OMERO server and save results back to the server.</p>
+
+<p>Further details on writing and installing scripts, the OMERO script repository and where to find other scripts are in <a href="http://www.openmicroscopy.org/site/community/scripts" target="_blank" class="external-url">Script Sharing</a> section  of the developer documentation.</p>
```

Comment:
The script sharing page isn't in the developer documentation, I'd just say "…are on the script sharing page."